clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -428,11 +428,19 @@ class MarkDeviceFunction : public RecursiveASTVisitor<MarkDeviceFunction> {
         Attrs.insert(A);
       else if (auto *A = FD->getAttr<ReqdWorkGroupSizeAttr>())
         Attrs.insert(A);
+      // Allow the following intel kernel attributes only on the lambda
```

Comment:
A series of rewording suggestions:
```suggestion
      // Allow the following kernel attributes only on lambda
```